### DISCUSSION TITLE
LAI assimilation without temporal smoother

### DISCUSSION THREAD

**Discussion by Shuzunyun** [NASA-LIS/LISF#1487] [Q&A] [answered]
Hi, LISF Community!

I am running Noah-MP v3.6 with the assimilation of GLASS LAI. However, when i set "GLASS LAI apply temporal smoother between 8-day intervals:" to be "0" (turn off temporal smoother), I get the following error message:

[DELL-PE-R750:1918539:0:1918539] Caught signal 8 (Floating point exception: floating-point invalid operation)
==== backtrace (tid:1918539) ====
 0  /lib64/libucs.so.0(ucs_handle_error+0x2a4) [0x146baa0dea54]
 1  /lib64/libucs.so.0(+0x26c2c) [0x146baa0dec2c]
 2  /lib64/libucs.so.0(+0x26eda) [0x146baa0deeda]
 3  ./LIS() [0xaba0fe]
 4  ./LIS() [0x810cab]
 5  ./LIS() [0x52eda1]
 6  ./LIS() [0xbc1c66]
 7  ./LIS() [0x8c1632]
 8  ./LIS() [0x411ae2]
 9  /lib64/libc.so.6(__libc_start_main+0xf3) [0x146bb34c9ca3]
10  ./LIS() [0x4119ee]
=================================
forrtl: error (75): floating point exception
Image              PC                Routine            Line        Source             
LIS                0000000000C11F9B  Unknown               Unknown  Unknown
libpthread-2.28.s  0000146BB3BE9CE0  Unknown               Unknown  Unknown
LIS                0000000000ABA0FE  noahmp36_updateve          87  noahmp36_updatevegvars.F90
LIS                0000000000810CAB  enkf_mod_mp_enkf_         642  enkf_Mod.F90
LIS                000000000052EDA1  lis_dataassimmod_         501  LIS_dataAssimMod.F90
LIS                0000000000BC1C66  retrospective_run         201  retrospective_runMod.F90
LIS                00000000008C1632  MAIN__                     68  lisdrv.F90
LIS                0000000000411AE2  Unknown               Unknown  Unknown
libc-2.28.so       0000146BB34C9CA3  __libc_start_main     Unknown  Unknown
LIS                00000000004119EE  Unknown               Unknown  Unknown

Does anyone know what the reason might be?

Looking forward to your reply!
Thanks!
Zunyun Shu

[nohup.out.txt](https://github.com/NASA-LIS/LISF/files/14126472/nohup.out.txt)
[run_da_LAI_2001.config.txt](https://github.com/NASA-LIS/LISF/files/14126473/run_da_LAI_2001.config.txt)

**Comment by dmocko** [comment]
Hi @Shuzunyun 

Can you also please copy your lislog file(s)?  I would like to see the end of these files to see if there is an error message or some other information.

If you turn on the smoother, does the simulation run as expected?

We usually have the smoother on, to prevent the model from regressing to unrealistic values between 8-day intervals.

**Reply by Shuzunyun** [reply, marked as answer]
Hi Dmocko,

Thank you for your reply. This problem was solved when I used the LISF-7.4.3.
